nemo_curator/utils/script_utils.py

```diff
@@ -13,335 +13,1276 @@
-        default=None,
-        help="Path to save dask profile",
-    )
-    return parser
-
-
-def parse_client_args(args: argparse.Namespace):
-    """
-    Extracts relevant arguments from an argparse namespace to pass to get_client
-    """
-    relevant_args = [
-        "scheduler_address",
-        "scheduler_file",
-        "n_workers",
-        "threads_per_worker",
-        "nvlink_only",
-        "protocol",
-        "rmm_pool_size",
-        "enable_spilling",
-        "set_torch_to_use_rmm",
-    ]
-    dict_args = vars(args)
-
-    parsed_args = {arg: dict_args[arg] for arg in relevant_args if arg in dict_args}
-    if "device" in dict_args:
-        parsed_args["cluster_type"] = dict_args["device"]
-
-    return parsed_args
-
-
-def parse_distributed_classifier_args(
-    description="Default distributed classifier argument parser",
-) -> argparse.ArgumentParser:
+
+
+class ArgumentHelper:
     """
-    Adds default set of arguments that are common to multiple stages
-    of the pipeline
+    A helper class to add common arguments to an argparse.ArgumentParser instance.
     """
 
-    parser = argparse.ArgumentParser(
-        description,
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
-    )
-    parser = add_distributed_args(parser)
-    # Set low default RMM pool size for classifier
-    # to allow pytorch to grow its memory usage
-    # by default
-    parser.set_defaults(rmm_pool_size="512MB")
-    parser.add_argument(
-        "--input-data-dir",
-        type=str,
-        help="The path of the input files",
-        required=True,
-    )
-    parser.add_argument(
-        "--output-data-dir",
-        type=str,
-        help="The path of the output files",
-        required=True,
-    )
-    parser.add_argument(
-        "--model-path",
-        type=str,
-        help="The path to the model file",
-        required=True,
-    )
-    parser.add_argument(
-        "--input-file-type",
-        type=str,
-        help="The type of the input files",
-        required=True,
-    )
-    parser.add_argument(
-        "--output-file-type",
-        type=str,
-        default="jsonl",
-        help="The type of the output files",
-        required=False,
-    )
-    parser.add_argument(
-        "--batch-size",
-        type=int,
-        default=128,
-        help="The batch size to be used for inference",
-    )
-    attach_bool_arg(
-        parser, "autocast", default=True, help_str="Whether to use autocast or not"
-    )
-    attach_bool_arg(
-        parser,
-        "enable-spilling",
-        default=True,
-        help_str="Whether to enable spilling or not",
-    )
-
-    # Setting to False makes it more stable for long running jobs
-    # possibly because of memory fragmentation
-    attach_bool_arg(
-        parser,
-        "set-torch-to-use-rmm",
-        default=False,
-        help_str="Whether to set torch to use RMM or not",
-    )
-
-    return parser
-
-
-def chunk_list(lst, nchnks):
```

Comment:
Were these functions deleted by mistake?

Reply:
Hi @ryantwolf , I searched for their usage, and there were not being used anywhere.